### DISCUSSION TITLE
Custom paths - best way to start?

### DISCUSSION THREAD

**Discussion by gsg-git** [mediagis/nominatim-docker#474] [Q&A]
Hello!
I have an ubuntu Server with aleready an classic Nominatim installation which doesnt run right now.
so i want to try the Docker version.

The System has 2 drives  in / is 500GB m2 and /home 2TB SATA SSD

The first try to set nominatim with Docker filled the M2 completely and stopped(full).

Is it enough to use WORKDIR at the end like this:

`docker run -it \
  -e PBF_URL=http://download.geofabrik.de/europe-latest.osm.pbf \
  -e REPLICATION_URL=https://download.geofabrik.de/europe-updates/ \
  -p 8080:5000 \
  --name nominatim \
  mediagis/nominatim:4.2 \
  WORKDIR /home/docker/Nominatim_Docker`

Or do i have to make a cusom image, because of all the other things happening in /app whil the setup or is temporarly 409152 MB enough?

I moved the Docker installation with symlinks to /home

Thank you!

**Comment by mtmail** [comment]
On Ubuntu by default Docker will create volumes in `/var/lib/docker/`.

The Nominatim docker image allows to put the output (the Postgres database, and the flatnode file) on separate volumes and thus path on your host machine. The remaining data (nominatim software, configuration files etc) can stay in the default `/var/lib/docker` path.

Have a look at the `-v` (short for `--mount`) setting in https://github.com/mediagis/nominatim-docker/tree/master/4.2#persistent-container-data

https://docs.docker.com/storage/volumes/ explains how volumes work, there's lots of introduction tutorials, too. I don't have a ready example but maybe another user can provide one.

For whole Europe you should use the flatnode-file feature (https://github.com/mediagis/nominatim-docker/tree/master/4.2#flatnode-files). Try the import with a small country first.

**Reply by gsg-git** [reply]
Thank you!
So the whole installation ist per default in the Docker folder(?).
I aleredy moved the Docker Folder to /home, because it was huge too so in now know why.
Im just wondering how the EU Files with around 27GB could create such a filesize bigger than 400GB ..

And im curios where the /app folder would be generated?
In the container or in the docker folder?
